test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -277,16 +320,56 @@ public void Sbom_Generation_Assigns_Correct_Verbosity_IgnoreCase(string inputVer
             PackageVersion = PackageVersion,
             NamespaceBaseUri = NamespaceBaseUri,
             Verbosity = inputVerbosity,
-            ManifestInfo = this.SbomSpecification.ToString(),
-            BuildEngine = this.buildEngine.Object
+            ManifestInfo = this.SbomSpecification,
+            BuildEngine = this.buildEngine.Object,
         };
 
         // Act
         var result = task.Execute();
         var output = stringWriter.ToString();
 
         // Assert
-        Assert.IsTrue(result);
-        Assert.IsTrue(pattern.IsMatch(output));
+        Assert.IsTrue(result, $"result: {result} is not set to true");
+        Assert.AreEqual(messageShouldBeLogged, pattern.IsMatch(output));
+    }
+#else
+    /// <summary>
+    /// Test to ensure GenerateSbom correctly parses and provides each EventLevel verbosity
+    /// values to the SBOM API.
```

Comment:
I would replace SBOM API with CLI Tool as this test is for the latter.

Reply:
Good catch. Changed at 1e50b1a3bb7c3df73bb2d192f3bdb7ad9ce7e40e